libraries/AP_Relay/AP_Relay.h

```diff
@@ -68,33 +68,37 @@ class AP_Relay {
 
     bool send_relay_status(const class GCS_MAVLINK &link) const;
 
+    // Set the state of all relays that are configured as the specified function type
```

Comment:
For the record, I find this mixed plurality functionality disconcerting. Here it's setting all relays that meet a filtering requirement, whereas the same method name with a different signature can also be used to set the state of a single relay instance (in the `private` section).

Ideally I think they should be renamed distinctly (e.g. this one could perhaps change to `set_all_by_function`), but I don't want to go digging through the codebase to find those different uses...